app/models/direct_file_data.rb

```diff
@@ -56,6 +56,10 @@ class DirectFileData
     primary_claim_as_dependent: 'IRS1040 PrimaryClaimAsDependentInd',
     hoh_qualifying_person_name: 'IRS1040 QualifyingHOHNm',
     surviving_spouse: 'IRS1040 SurvivingSpouseInd',
+    third_party_designee: 'IRS1040 ThirdPartyDesigneeInd',
```

Comment:
it might be nice to keep the "ind" / "indicator" in the name to make it more clear that this is the boolean value not the name (out of context it can be hard to keep it straight). suggesting `third_party_designee_ind` or `third_party_designee_indicator`